doc/troubleshooting.md

```diff
@@ -153,8 +193,41 @@ Events:                   <none>
 
 ## Logs missing when Pravega upgrades
 
-Users may find Pravega logs for old pods to be missing post upgrade. This is because the operator uses the Kubernetes 
+Users may find Pravega logs for old pods to be missing post upgrade. This is because the operator uses the Kubernetes
 [rolling update](https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/#updating-statefulsets)
-strategy to upgrade pod one at a time. This strategy will use a new replicaset for the update, it will kill one pod in the 
+strategy to upgrade pod one at a time. This strategy will use a new replicaset for the update, it will kill one pod in the
 old replicaset and start a pod in the new replicaset in the meantime. So after upgrading, users are actually using a new
 replicaset, thus the logs for the old pod cannot be obtained using `kubectl logs`.
+
+## Operator pod in container creating state
+
+while installing operator, if the operator pod goes in `ContainerCreating` state for long time, make sure certificates are installed correctly.Please refer [prerequisite](../charts/pravega-operator/README.md#Prerequisites)
+
+## Recover Operator when node fails
+
+If the Operator pod is deployed on the node that fails, the pod will be rescheduled to a healthy node. However, the Operator will
+not function properly because it has a leader election locking mechanism. See [here](https://github.com/operator-framework/operator-sdk/blob/master/doc/proposals/leader-for-life.md).
+
+To make it work, the cluster admin will need to delete the lock by running
+
+```
+kubectl delete configmap pravega-operator-lock
+```
+After that, the new Operator pod will become the leader. If the node comes up later, the extra Operator pod will
+be deleted by Deployment controller.
+
+## Collecting log files for crashed pod
+
+For collecting log files for crashed pod, use the below command
+
+```
+kubectl logs <podname> --previous
+```
+
+## Filtering kubectl Events
```

Comment:
This heading is not part of the contents mentioned above as part of the Pravega Cluster / Pravega Operator issues. Also could we arrange the sections in the same sequence in which they are listed in the contents?